upgrade_analysis/models/upgrade_comparison_config.py

```diff
@@ -43,11 +43,11 @@ def get_connection(self):
         self.ensure_one()
         try:
             remote = odoorpc.ODOO(self.server, port=self.port)
-        except URLError:
+        except URLError as exc:
             raise UserError(
-                _("Could not connect the Odoo server at %s:%s")
-                % (self.server, self.port)
-            )
+                _("Could not connect the Odoo server at %(server)s:%(port)s")
+                % {"server": self.server, "port": self.port}
+            ) from exc
```

Comment:
```suggestion
            ) 
```

Reply:
I don't get it. could you elaborate ? I introduced this changes because of error in the pre-commit checks. 
The message was talking about adding from, and was giving the link : https://docs.python.org/3/tutorial/errors.html#exception-chaining

thanks.

Reply:
I don't get it. could you elaborate ? I introduced this changes because of error in the pre-commit checks.
The message was talking about adding from, and was giving the link : https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
Interesting, thanks. I thought it was needed only if exc was used.